config/locales/find.yml

```diff
@@ -209,6 +209,11 @@ en:
           uk_citizens: UK citizens
           non_uk_citizens: Non-UK citizens
           course_fees: The course fees for UK citizens in %{cycle_range} are %{fee}.
+      tda_advice:
+        heading: Teacher degree apprenticeships (TDAs)
+        content: On a teacher degree apprenticeship you’ll work in a school and earn a salary while getting a bachelor’s degree and qualified teacher status (QTS).
+        link_text: Find out more about teacher degree apprenticeships.
+        link_url: https://getintoteaching.education.gov.uk/train-to-be-a-teacher/teacher-degree-apprenticeships
```

Comment:
Same with above.
There is one component that uses. Maybe we can move this translations around?
```
       <%= t(                                                                     
         ".tda_with_qts_content_html",                                            
         tda_link: govuk_link_to(t(".find_out_more_about_tda"), t("find.get_into_teaching.url_tda"))
       ) %> 
```